src/RadialBarChart/RadialBarSeries/MotionBar.tsx

```diff
@@ -8,26 +8,26 @@ export const MotionBar = ({ custom, transition, arc, ...rest }) => {
   const currentYRef = useRef(custom.exit.y);
   const spring = useSpring(0, {
     ...DEFAULT_TRANSITION,
-    delay: transition.delay,
-    from: 0,
-    to: 1
+    delay: transition.delay
   });
 
-  useEffect(() => {
-    spring.set(1);
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
-
   useEffect(() => {
     const interpolator = interpolate(currentYRef.current, custom.enter.y);
+    const prevSpring = spring.get();
 
-    spring.set(1);
+    if (transition?.delay) {
+      setTimeout(() => {
```

Comment:
Do we want to clear the timeout in the event it gets multiple queued?

Reply:
oh yea good call